app/workers/fast_track/hypertension_pdf_generator.rb

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionPdfGenerator
+    attr_accessor :patient, :bp_data, :medications
+
+    def initialize(patient, bp_data, medications, date)
+      @patient = patient
+      @bp_data = bp_data
+      @medications = medications
+      @date = date
+    end
+
+    def generate
```

Comment:
Are there steps I could take in the review instance to generate this PDF?

Reply:
It relies on test data to be in place in Lighthouse. We're currently planning on performing a functional test with two different Lighthouse-connected patients that are set up to qualify for our fast track process. We are still waiting on Lighthouse data to be available, however. If you'd like, I could set up a meeting for us all to walk through it together?

Reply:
I'm trying to understand. Are you saying the only way to test this is in staging or production?

Reply:
The PDF is only generated if the Lighthouse data has sufficient BP readings from the last year.
The generated PDF is in-memory and written to S3 via SupportingEvidenceAttachment and eventually ends up with any files the user themselves uploaded as part of the Form 526 submission.
In order to write the PDF to a different location, such as a file, in order to view it, I think you'd have to have access to the Rails console, call `pdf.render_file` to some path, and then view that path.

Reply:
Sorry in advance if I'm not answering your question, but to add more detail:

What I know for sure is that we need our test Lighthouse data which is also connected to a test `va.gov` user in order to test that everything works, including that the PDF looks as it's supposed to. We don't know of a way to do this completely locally - at least, not without taking a long time to set up test instances of every service with test data - maybe there are established patterns for doing that we're not aware of. 

So far, with `vets-api` running locally and pointing to staging, we've been able to test the generated PDF and the S3 upload (the S3 upload is configured to upload to a test AWS instance, though - not within VA private cloud) with test patients in staging. We now have our Lighthouse staging data that will allow us to test the entire flow, including kicking off the Sidekiq job for the *qualifying* test user and pushing the generated PDF along with other uploaded files to EVSS.

Let me know if you want more clarity - the offer to meet about this stands, and feel free to ping me about setting it up.